themes/atvp/css/imports/content-components.css

```diff
@@ -61,3 +61,51 @@
 		}
 	}
 }
+
+/*--------------------------------------------------------------
+	# Blocks
+--------------------------------------------------------------*/
+
+/* --- Buttons --- */
+.wp-block-buttons {
+
+	.wp-block-button {
+		margin: var(--spacing-050) clamp(var(--spacing-2), 10vw, var(--spacing-6)) var(--spacing-050) 0;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+
+	/*stylelint-disable */
+	.is-style-arrow-link .wp-block-button__link {
+		background-color: transparent;
+		border: none;
+		color: var(--color-alert);
+		display: flex;
+		font-size: var(--font-size-medium);
+		font-weight: 700;
+		padding: 0;
+
+		&:hover {
+			background-color: transparent;
+			text-decoration: underline;
+		}
+
+		&::after {
+			content: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 14 14' xmlns='http://www.w3.org/2000/svg'%3E%3Ctitle%3E→%3C/title%3E%3Cpath d='M6.692 13.432L13.412 7v-.096L6.692.472 4.772 2.68l3.12 2.64H.068v3.264h7.824l-3.12 2.64z' fill='%23BF231B' fill-rule='nonzero'/%3E%3C/svg%3E");
```

Comment:
We can probably remove the `%3Ctitle%3E→%3C/title%3E` bit here and below - I don't _think_ `<title>...</title>` matters much in this context?